pkg/dependency/parser/java/pom/pom.go

```diff
@@ -73,6 +76,26 @@ func (p pom) projectProperties() map[string]string {
 	return projectProperties
 }
 
+// Get properties from any maven profiles, only for now if one is active by default
+func (p pom) profileProperties() map[string]string {
+	profiles := p.content.Profiles
+	logger := log.WithPrefix("pom")
+
+	projectProperties := make(map[string]string)
+	for _, profile := range profiles.Profile {
+		activation := profile.Activation
+		if activation.ActiveByDefault == enabled {
+			logger.Debug("Adding properties from profile", log.String("id", profile.ID))
```

Comment:
nit:
We may not create a new variable every time
```suggestion
	projectProperties := make(map[string]string)
	for _, profile := range p.content.Profiles.Profile {
		if profile.Activation.ActiveByDefault == enabled {
			log.WithPrefix("pom").Debug("Adding properties from profile", log.String("id", profile.ID))
```